CHANGELOG.md

```diff
@@ -47,7 +48,6 @@
 
 ### Changes
 - Updated translations: `pa-IN`, `hi-IN`, `te-IN`, `kn-IN`, `mr-IN`, `te-IN`, `sid-ET`, `kn-IN`, `ta-IN`, `bn-BD`, `bn_IN`, `so-ET`, `ti-ET`, `am-ET`, `ta-LK`, `om-ET`
-- New supported language: Tamil (Sri Lanka) [`ta-LK`]
```

Comment:
Umm. I was wondering that why did we have this in the changelog before if we just added the support..

Reply:
We added that when we initially merged the translated strings, usually we add the language option pretty soon. But here we didn't. So moved it to the next release section.